src/main/resources/admin/tools/vue/util/rawSchemas/index.ts

```diff
@@ -0,0 +1,55 @@
+import AttachmentUploader from './schemas/attachment-uploader';
```

Comment:
More of a future architecture/design note:
I don't see the reason to interpret a xml -> json just because we want to create a new input type (element) 
Cant we define the rawSchema as json? Would save a lot of xml2js calls.